apps/prairielearn/python/test/check_data_test.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-from check_data import check_data
+from prairielearn.internal.check_data import check_data
```

Comment:
I would like to either use `internal.check_data` or `_check_data`, if someone has a preference for the latter, I can change it.